index.html

```diff
@@ -2,237 +2,257 @@
             </ul>
-            <div class="action">
-              <button class="btn">See Project</button>
-            </div>
           </div>
         </div>
-      
-        <div class="section5">
-          <div class="top">
-            <h3 class="about-heading">About <br>Myself</h3>
-            <p class="about-description">I'm a software developer! I can help you build a product, feature or website. Look
-              through some of my work and experience. If you like what you see and have a project you need coded, don't
-              hestiate to contact me.</p>
-            <h3 class="social order2">LET'S CONNECT</h3>
-            <ul class="social-links">
-              <li><a href="#"><i class="fa-brands fa-twitter li1"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-linkedin li2"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-github li4"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
-            </ul>
-            <button class="resume">Get my resume</button>
-          </div>
-          <div class="skills-list">
-            <div class="languages">
-              <div class="container">
-                <span class="lang">Languages</span>
-                <i class="fa-solid fa-angle-down btn1"></i>
-              </div>
-              <div class="language-list">
-                <ul>
-                  <li class="l-list"><img class="images" src="image/EllipseJS.png"><span
-                      class="js">Javascript</span></li>
-                  <li class="l-list"><img class="images" src="image/EllipseHTML.png"><span
-                      class="js">HTML</span></li>
-                  <li class="l-list"><img class="images" src="image/EllipseCSS.png"><span class="js">CSS
-                    </span></li>
-                </ul>
-              </div>
-            </div>
-            <div class="container">
-              <span class="lang">Frameworks</span>
-              <i class="fa-solid fa-angle-right btn1"></i>
-            </div>
-            <div class="container">
-              <span class="lang">Skills</span>
-              <i class="fa-solid fa-angle-right btn1"></i>
-            </div>
-          </div>
+        <div class="container">
+          <span class="lang">Frameworks</span>
+          <i class="fa-solid fa-angle-right btn1"></i>
+        </div>
+        <div class="container">
+          <span class="lang">Skills</span>
+          <i class="fa-solid fa-angle-right btn1"></i>
         </div>
-      </section>
-  
+      </div>
+    </div>
+  </section>
+  <section class="contact-form">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that
+      needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] You did a wonderful job adding `action` and `method` attributes :+1:  but I've noticed that you didn't change `{form_id}` inside the link in `action`, so it is not sending information. You should register on [formspree service](https://formspree.io/html) and then you can get the required id to replace `{form_id}` :+1: